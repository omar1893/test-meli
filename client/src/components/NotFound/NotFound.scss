.notFoundContainer {
    text-align: center;
    h2 {
        font-size: 4.2rem;
    }
    p {
        font-size: 3rem;
    }
}