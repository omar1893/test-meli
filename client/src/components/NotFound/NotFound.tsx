import React from 'react'
import './NotFound.scss'

const NotFound = () => {
  return (
    <div className='notFoundContainer'>
      <h2>Oops, lo sentimos...</h2>
      <p>No pudimos encontrar ningún producto relacionado con tu búsqueda.</p>
    </div>
  )
}

export default NotFound