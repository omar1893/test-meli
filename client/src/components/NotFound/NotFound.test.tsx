import React from 'react';
import { render, screen } from '@testing-library/react';
import NotFound from './NotFound';

describe('NotFound component', () => {
    it('should match snapshot', () => {
      const { container } = render(<NotFound />);
      expect(container).toMatchSnapshot();
    });

    it('should renders correct info', () => {
        render(<NotFound />);
        const headingElement = screen.getByText("Oops, lo sentimos...");
        const paragraphElement = screen.getByText("No pudimos encontrar ningún producto relacionado con tu búsqueda.");
        
        expect(headingElement).toBeInTheDocument();
        expect(paragraphElement).toBeInTheDocument();
      });

})