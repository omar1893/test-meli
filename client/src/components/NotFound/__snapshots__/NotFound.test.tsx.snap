// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound component should match snapshot 1`] = `
<div>
  <div
    class="notFoundContainer"
  >
    <h2>
      Oops, lo sentimos...
    </h2>
    <p>
      No pudimos encontrar ningún producto relacionado con tu búsqueda.
    </p>
  </div>
</div>
`;
